pkg/virtctl/hotplug/common/common.go

```diff
@@ -0,0 +1,36 @@
+/*
+ * This file is part of the KubeVirt project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Copyright 2024 The KubeVirt Contributors
+ *
+ */
+
+package common
+
+import (
+	"context"
+	"time"
+
+	k8smetav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"kubevirt.io/client-go/kubecli"
+)
+
+const RequestTimeout = 10 * time.Second
+
+func ValidateVirtualMachineExists(ctx context.Context, virtCli kubecli.KubevirtClient, namespace, name string) error {
```

Comment:
I would drop this helper func altogether. It simply wraps a `Get` call.

Reply:
yea you are right, I ended up dropping the common package completely